### DISCUSSION TITLE
Popular cryptographic algorithms performance in OpenSSL

### DISCUSSION THREAD

**Discussion by quicmmarimut** [actions/virtual-environments#11250] [General questions]
Hi Team,
    We wanted to understand the list of popular cryptographic algorithms and what kind of performance profiling is recommended for them. Below is a short list of algorithms I have chosen to profile the performance on an ARM64 device (Windows on ARM). can you suggest any other algorithm based on popularity or any telemetry data recorded ?

Do we have any place where we can get to know the telemetry data on these algorithms usage and identify the popularity based on that ?

List of algorithms I have chosen:
<html xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=PowerPoint.Slide>
<meta name=Generator content="Microsoft PowerPoint 15">
<style>

</style>
</head>

<body>

S. No | Algorithm Type | Algorithm
-- | -- | --
1 | Symmetric Ciphers | AES-256-CBC
2 | Symmetric Ciphers | DES-EDE3-CBC
3 | Symmetric Ciphers | ChaCha20
4 | Asymmetric Ciphers | RSA
5 | Asymmetric Ciphers | DSA
6 | Asymmetric Ciphers | ECDSA
7 | Hash Functions | SHA-256
8 | Hash Functions | SHA-512
9 | Hash Functions | MD5
10 | Hash Functions | SHA-3
11 | Key Exchange | DH
12 | Key Exchange | ECDH
13 | Message Authentication Codes (MACs) | HMAC-SHA256
14 | Message Authentication Codes (MACs) | CMAC-AES

</body>

</html>

Thanks in advance!
 Mallikarjunan

**Comment by subir0071** [comment]
Hi @quicmmarimut  - Unfortunately,this topic is beyond the scope of this repository. 
Thanks.
